### PR TITLE
DBFlow dependency refactor

### LINKED ISSUE #1878: Build fails due to a dependency on DBFlow
**Summary:** 
Since one of the libraries in the project build is throwing the following error

> Execution failed for task ':mifosng-android:kaptGenerateStubsDebugKotlin'.
> Could not resolve all files for configuration ':mifosng-android:kapt'.
> Could not find dbflow-core.jar (com.github.Raizlabs.DBFlow:dbflow-core:3.1.1).

**Steps to reproduce:** 
Open the project in the latest version of Android Studio and build the application.
How can we reproduce the issue?

**Expected behaviour:** 
After cloning, the app should build successfully without any issues

**Screenshots:** 

image

### PR DESCRIPTION
**Pull request by avinash14022002**
Fixes #1878 

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `MifosStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

File: mifosng-android/build.gradle [hunk 1]
```diff
@@ -156,12 +156,12 @@ dependencies {
 
     //DBFlow dependencies
     implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
-    annotationProcessor "com.github.Raizlabs.DBFlow.dbflow:dbflow-processor:3.1.1"
-    annotationProcessor "com.github.Raizlabs.DBFlow:dbflow-processor:3.1.1"
+    annotationProcessor "com.github.raizlabs.dbflow.dbflow:dbflow-processor:3.1.1"
```
Comment (danishjamal104) [inline mifosng-android/build.gradle:159]: what is the reason of renaming this?

File: mifosng-android/build.gradle [hunk 2]
```diff
@@ -156,12 +156,12 @@ dependencies {
...
+    annotationProcessor "com.github.raizlabs.dbflow:dbflow-processor:3.1.1"
     //annotationProcessor "com.github.Raizlabs.DBFlow.dbflow-core:dbflow-processor:$rootProject.raizLabsDBFlow"
     //implementation "com.github.Raizlabs.DBFlow:dbflow-core:$rootProject.raizLabsDBFlow"
-    implementation "com.github.Raizlabs.DBFlow.dbflow:dbflow:$rootProject.raizLabsDBFlow"
-    kapt "com.github.Raizlabs.DBFlow:dbflow-processor:3.1.1"
+    implementation "com.github.raizlabs.dbflow.dbflow:dbflow:$rootProject.raizLabsDBFlow"
+    kapt "com.github.raizlabs.dbflow:dbflow-processor:3.1.1"
```
Comment (danishjamal104) [inline mifosng-android/build.gradle:164]: can you provide a reference for the dbFlow dependency name update?

File: mifosng-android/build.gradle (hunk 1 above)
Comment (avinash14022002) [reply mifosng-android/build.gradle:159]: @danishjamal104 
[Link ](https://github.com/agrosner/DBFlow/issues/1321#issuecomment-307728901) for the reference

Comment (danishjamal104) [reply mifosng-android/build.gradle:159]: cool

**Review by danishjamal104 — APPROVED** [review]
LGTM

**Comment by danishjamal104** [comment]
@iamsh4shank merge this. Thanks :)

File: mifosng-android/build.gradle (hunk 2 above)
Comment (cjxonix) [reply mifosng-android/build.gradle:164]: This worked well I had the same issue.

**Pull request merged**